src/upgrade-compatibility-tool/install.md

```diff
@@ -0,0 +1,123 @@
+1. The Software Engineer runs the Upgrade Compatibility Tool one more time with a Magento pre-release to ensure there are no new issues and that their code changes fixed the problems found during the beta phase.
+1. Everything check’s out and the Software Engineer pushes the code to a staging environment where regression tests confirm all tests are green, which allows them to release the latest Magento version to production the same day that the Magento pre-release is released.
+
+![Upgrade Compatibility Tool audience](img/audience-uct.png)
+
+### Contact Upgrade Compatibility Tool
+
+To connect with the Upgrade Compatibility Tool team:
+
+1. Contact us on the Engineering Slack channel [Magento Upgrade Compatibility Tool](https://magentocommeng.slack.com/archives/C019Y143U9F).
+1. Send us an email at [uct@adobe.com](mailto:uct@adobe.com).
+
+See the [Resources]({{site.baseurl}}/community/resources/resources.html) page for more information.
+
+## Prerequisites
+
+See [prerequisites]({{site.baseurl}}/upgrade-compatibility-tool/prerequisites.html) for more information.
+
+{:.bs-callout-info}
+You can run the Upgrade Compatibility Tool in any operating system. There is no requirement to run the Upgrade Compatibility Tool where your Magento instance is located. It is necessary for the Upgrade Compatibility Tool to have access to the source code of the Magento instance. For example, you can install the tool on one server and point it at your Magento installation on another server.
+
+If you are running the Upgrade Compatibility Tool against a Magento instance with large modules and files, the tool might require a high amount of RAM, at least 2GB RAM.
+
+### Recommended actions
+
+Magento best practice is not to have 2 modules with the same name, if this happens the Upgrade Compatibility Tool will show a segmentation fault error in which case you have to analyze each module independently with the option `-m`:
+
+```bash
+bin/uct upgrade:check /(instance_path) --coming-version=2.4.1 -m /(module_path)
+```
+
+If you get memory issues while executing the Upgrade Compatibility Tool it is recommended to use the `-m` command to run the tool against a specific module.
+
+## Download the Upgrade Compatibility Tool
+
+To download the Upgrade Compatibility Tool, run the following command:
+
+```bash
+composer create-project magento/upgrade-compatibility-tool uct  --repository https://repo.magento.com
+```
+
+As the Upgrade Compatibility Tool is an independent tool, if you try to run:
+
+```bash
+composer require magento/upgrade-compatibility-tool
+```
+
+It might add the Upgrade Compatibility Tool as a dependency for a Magento project.
+
+## Install
+
+To install the Upgrade Compatibility Tool, you must install the necessary prerequisites:
+
+*  Magento access keys
+*  Composer
+*  Node.js (only required to check GraphQL compatibility)
+
+Refer to the [Upgrade Compatibility Tool install]({{site.baseurl}}/upgrade-compatibility-tool/install.html#install) page.
```

Comment:
Same as above, repo name has not changed yet

Reply:
same comment as per the docs folder in the devdocs github repo